### ISSUE TITLE
Az.VMware does not load on PowerShell for Linux

### ISSUE THREAD

**Issue by ctaggart** [Azure/azure-powershell#14434]
## Description
Using PowerShell on Linux (via Docker or Azure Cloud Shell) , Az.VMware does not work. I can not get the commands for the module after it is installed. I'm expecting this to work like this output on Windows:

``` powershell
Get-Command -Module Az.VMWare

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Function        Get-AzVMwareAuthorization                          0.2.0      Az.VMware
Function        Get-AzVMwareCluster                                0.2.0      Az.VMware
Function        Get-AzVMwarePrivateCloud                           0.2.0      Az.VMware
Function        Get-AzVMwarePrivateCloudAdminCredentials           0.2.0      Az.VMware
Function        New-AzVMwareAuthorization                          0.2.0      Az.VMware
Function        New-AzVMwareCluster                                0.2.0      Az.VMware
Function        New-AzVMwarePrivateCloud                           0.2.0      Az.VMware
Function        Remove-AzVMwareAuthorization                       0.2.0      Az.VMware
Function        Remove-AzVMwareCluster                             0.2.0      Az.VMware
Function        Remove-AzVMwarePrivateCloud                        0.2.0      Az.VMware
Function        Test-AzVMwareLocationQuotaAvailability             0.2.0      Az.VMware
Function        Test-AzVMwareLocationTrialAvailability             0.2.0      Az.VMware
Function        Update-AzVMwareCluster                             0.2.0      Az.VMware
Function        Update-AzVMwarePrivateCloud                        0.2.0      Az.VMware
```

## Steps to reproduce

```powershell
docker run --rm -it mcr.microsoft.com/azure-powershell
PowerShell 7.1.2
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS /> Install-Module -Name Az.VMware

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its
InstallationPolicy value by running the Set-PSRepository cmdlet. Are you sure you want to install the modules from
'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): A
PS /> Get-Command -Module Az.VMware
PS /> Get-Command -Module Az.VMWare
```

## Environment data

```
docker images
REPOSITORY                           TAG       IMAGE ID       CREATED        SIZE
mcr.microsoft.com/azure-powershell   latest    787301fe4b38   2 days ago     526MB
```

## Module versions

```powershell

```

## Debug output

```

```

## Error output

```

```

**Comment by ctaggart** [comment]
This appears to have been fixed by #14433.

```
PS /> Install-Module Az.VMware

Untrusted repository
You are installing the modules from an untrusted repository. If you trust this repository, change its InstallationPolicy value by running the
Set-PSRepository cmdlet. Are you sure you want to install the modules from 'PSGallery'?
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "N"): A
PS /> Get-Command -Module Az.VMware

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Function        Get-AzVMwareAuthorization                          0.2.0      Az.VMware
Function        Get-AzVMwareCluster                                0.2.0      Az.VMware
Function        Get-AzVMwarePrivateCloud                           0.2.0      Az.VMware
Function        Get-AzVMwarePrivateCloudAdminCredentials           0.2.0      Az.VMware
Function        New-AzVMwareAuthorization                          0.2.0      Az.VMware
Function        New-AzVMwareCluster                                0.2.0      Az.VMware
Function        New-AzVMwarePrivateCloud                           0.2.0      Az.VMware
Function        Remove-AzVMwareAuthorization                       0.2.0      Az.VMware
Function        Remove-AzVMwareCluster                             0.2.0      Az.VMware
Function        Remove-AzVMwarePrivateCloud                        0.2.0      Az.VMware
Function        Test-AzVMwareLocationQuotaAvailability             0.2.0      Az.VMware
Function        Test-AzVMwareLocationTrialAvailability             0.2.0      Az.VMware
Function        Update-AzVMwareCluster                             0.2.0      Az.VMware
Function        Update-AzVMwarePrivateCloud                        0.2.0      Az.VMware
```

**Comment by ctaggart** [comment]
I'm confirming that #14433 did fix this for Azure Cloud Shell as well.  To fix, you may need to uninstall the current module first.
```
Uninstall-Module Az.VMware
Install-Module Az.VMware
```

![image](https://user-images.githubusercontent.com/80104/110188320-168de680-7dd0-11eb-8d35-de497650140f.png)
